src/App.js

```diff
@@ -0,0 +1,106 @@
+import React, {Component} from 'react';
+import _ from 'lodash'
+
+// import logo from './logo.svg';
+import FiveCardLayout from './components/FiveCardLayout.js'
+
+import CARDS_DATA from './cards.json'
+import './App.css'
+
+import LAYOUT_DATA from './layoutData.json'
+
+const LAYOUTS = {
+  'FiveCard': FiveCardLayout,
+}
+
+class App extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      layout: 'FiveCard',
+      cards: this.pickCards(),
+      flippedIndex: 0,
+      positionTitle: '',
+      positionInfo: '',
+      question: '',
+      questionEntered: false
+    }
+  }
+
+  pickCards(){
+    return _.sampleSize( CARDS_DATA, 5 ).map( card =>
+      Object.assign( {}, card, { faceUp: false, isReversed: !!_.random(1) } )
+    )
+  }
+
+  flipCard() {
+    const { cards, flippedIndex } = this.state
+
+    this.setState({
+      cards: [
+        ...cards.slice( 0, flippedIndex ),
```

Comment:
agreed. It seems like you could just get `this.state.cards` find the card you need to change, change it, and then `this.setState({cards})`